File: homafo_backend/src/main/java/com/bioagricola/aforos/entity/dto/MaestroVisitasResource.java
package com.bioagricola.aforos.entity.dto;

import java.util.Date;
import java.util.List;


import lombok.Data;


@Data
public class MaestroVisitasResource {

	private Long mafvIderegistro;
	private Date mafvInicio;
	private Date mafvFin;
	private String mafvEstado;
	private Date mafvFecharegistro;
	private Date mafvFechaactualizacion;
	private Long perIderegistro;
	private Long cicCiclo;
	private Long uniTipogenerador;
	private String mafvFactor;
	private Long usuIderegistro;
	private Long perIderegistrofin;
	private List<DetalleMaestroVisitaResource> detallesMaestroVisita;
	private Long mafvMinimoVisitas;
	private Long aforo;

}

File: homafo_frontend/src/models/dto/MaestroVisitasResource.ts
import DetalleMaestroVisita from "./DetalleMaestroVisita";

export default interface MaestroVisitasResource {
    mafvideregistro:number;
	mafvinicio:Date;
	mafvfin:Date;
	mafvestado:string;
	mafvfecharegistro:Date;
	mafvfechaactualizacion:Date;
	perideregistro:number;
	cicciclo:number;
	unitipogenerador:number;
	mafvfactor:string;
	usuideregistro:number;
	perideregistrofin:number;
	detallesMaestroVisita?:DetalleMaestroVisita[];
	mafvminimovisitas:number;
	aforo:number;
}